frontend/components/site/SideNav.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+
+const SideNav = ({ siteId }) => (
```

Comment:
simplified this component by just listing the links explicitly instead of via a config and child components